src/types/channels.ts

```diff
@@ -20,6 +20,7 @@ export type Channel = {
     update_at: number;
     delete_at: number;
     team_id: string;
+    team_display_name: string;
```

Comment:
should this be a required field?

Reply:
Alternatively, should this be a separate type that extends `Channel`? I don't think most of the APIs return channels with team information included